module/zfs/zpl_inode.c

```diff
@@ -774,6 +813,7 @@ const struct inode_operations zpl_dir_inode_operations = {
 	.permission	= zpl_permission,
 #endif /* HAVE_GET_ACL | HAVE_CHECK_ACL | HAVE_PERMISSION */
 #endif /* CONFIG_FS_POSIX_ACL */
+	.fiemap		= zpl_fiemap,
```

Comment:
This provides extent maps for directories, correct?

Reply:
Yes.